Tools/setup/ubuntu.sh

```diff
@@ -203,6 +203,10 @@ if [[ $INSTALL_SIM == "true" ]]; then
 		protobuf-compiler \
 		;
 
+	# FlightGear
+	sudo add-apt-repository -y -u ppa:saiarcot895/flightgear
+	sudo DEBIAN_FRONTEND=noninteractive apt-get -y --quiet install flightgear
```

Comment:
In my experience, Flight gear takes quite some time to install, and I don't think this should be the default. Could we separate this out?

Reply:
ok, what is the best way to separate flightgear installation?

Reply:
@Jaeyoung-Lim @kaklik this is still unanswered, and I agree, it should be optional / a separate script.

Reply:
I have removed it from the script. Installation will be explained in the Devguide documentation.